lib/commands/connect/pause.js

```diff
@@ -14,19 +14,10 @@ module.exports = {
   needsApp: true,
   needsAuth: true,
   run: cli.command(co.wrap(function* (context, heroku) {
-    let connection = yield api.withConnection(context, heroku);
-    let url = '/api/v3/connections/' + connection.id + '/actions/pause';
-    let response = yield api.request(context.auth.password, 'POST', url);
-
-    switch(response.statusCode) {
-      case 202:
-        console.log(api.connection_string(connection) + ' paused');
-      break;
-      case 409:
-        console.log(response.json['message']);
-      break;
-      default:
-        cli.debug(response);
-    }
+    yield cli.action('pausing connection', co(function* () {
+      let connection = yield api.withConnection(context, heroku);
+      let url = '/api/v3/connections/' + connection.id + '/actions/pause';
+      yield api.request(context.auth.password, 'POST', url);
```

Comment:
What's yielded here is irrelevant. This doesn't work like Python where yield sends a value back to the caller. It's just to yield control to something else asynchronously. So in case of success, we just get "done", and in the case of error, the action will display the content of the error object that was thrown. In this case, any API response >= 400 will indeed throw an error with `response.json['message']`